src/github.com/getlantern/flashlight/server/server.go

```diff
@@ -295,6 +295,11 @@ func (server *Server) checkForBannedCountry(req *http.Request) error {
 }
 
 func (server *Server) lookupCountry(req *http.Request) (string, error) {
+	// Use the country CloudFlare gives us if it's available.
+	cf := req.Header.Get("Cf-Ipcountry")
```

Comment:
The main reason I ask this is because I was thinking on potential issues with the encoding used for countries in CF-IPCountry. Are we confident both Cloudflare and us are using the same format? Cloudflare is using ISO 3166-1 Alpha 2 format, I checked that.